# from django.core.exceptions import ValidationError

# def validate_close_prof(self):
#     if self.fromUser - self.toUser > 2:
#         raise ValdationError("Sorry please pick a penpal with closer proficiency")
#     elif self.fromUser - self.toUser < -2:
#         raise ValdationError("Sorry please pick a penpal with closer proficiency")  
    
